features/token.feature

```diff
@@ -0,0 +1,13 @@
+@javascript
+Feature: Token Generation
+
+Scenario: As a survivor, I want to have a printed token
+Given a basic token layout is defined
+And I am registered in the database using
+| Attribute | Value       |
+| id        | 1           |
+| input2-1  | Input 2 - 1 |
+| input2-2  | Input 2 - 2 |
+| input3-1  | Input 3 - 1 |
+| input3-2  | Input 3 - 2 |
+Then I should see a token
```

Comment:
capybara doesn't handle file download and the people_controller generates the token on the fly. Changed to "Then I can generate and see a token". There may be a use case to change the floor of people_controller. Can update the test then.